### PR TITLE
docs: new nav

### PR DESCRIPTION
**Pull request by viceice**
## Changes
- add new nav
- add datasource metadata

## Context
- https://github.com/renovatebot/renovatebot.github.io/pull/253

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Comment by viceice** [comment]
ready for review, will probably cause a failure on docs repo until the other pr get's merged

File: docs/usage/modules/manager/.pages [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title: Managers
```
Comment (HonkingGoose) [inline docs/usage/modules/manager/.pages:1]: ### Weird order in sidebar for manager intro page

I'm checking out the preview on the Renovate docs PR and the sidebar is confusing:

manager-intro-out-of-order

The page title is "Introduction", and it's in the wrong place in the sidebar.

### Order in which Renovate executes

I think Renovate follows this ordering in the execution of the program:

1. Platforms
2. Managers
3. Datasources
4. Versioning

### Correct sidebar idea

We should probably match the order given above in the docs.

Maybe the title for the `managers.md` needs to be changed in the `docs/usage/modules/manager/index.md` file?

File: docs/usage/.pages [hunk 2]
```diff
@@ -0,0 +1,51 @@
+nav:
+  - Home: 'index.md'
+  - ... | getting-started
+  - Troubleshooting: 'troubleshooting.md'
+  - Configuration:
+      - 'Self-hosted': 'self-hosted-configuration.md'
+      - 'Repository': 'configuration-options.md'
+      - 'Presets': 'config-presets.md'
+  - ... | key-concepts
+  - ... | modules
```
Comment (HonkingGoose) [inline docs/usage/.pages:10]: On the preview I can't see a ⬇️ to expand the Datasources sidebar.

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

File: docs/usage/modules/manager/.pages (hunk 1 above)
Comment (viceice) [reply docs/usage/modules/manager/.pages:1]: 👉 
image
natural sort order

File: docs/usage/.pages (hunk 2 above)
Comment (viceice) [reply docs/usage/.pages:10]: yes, it's a single page needs this 😉 
- #14772

File: docs/usage/modules/manager/.pages [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+nav:
+  - Introduction: index.md
+# TODO: add all managers ?
+#  - ...
```
Comment (viceice) [inline docs/usage/modules/manager/.pages:5]:
```suggestion
  - index.md
# TODO: add all managers ?
#  - Introduction: index.md
#  - ...
```

**Review by HonkingGoose — APPROVED** [review]
I guess this is good to go now that my review comments are resolved.

**Review by rarkins — APPROVED** [review]

**Pull request merged**